pygameMenu/widgets/textinput.py

```diff
@@ -35,6 +35,27 @@
 from pygameMenu import locals as _locals
 from pygameMenu.widgets.widget import Widget
 
+try:
+    from pyperclip import copy, paste
+except ModuleNotFoundError:
+    def copy():
```

Comment:
signature of the function shall be the same has the one from `pyperclip.copy` (take one argument).

Reply:
Oops, my bad 😭. Push any pull request if you wish, I will be abroad for a couple of days and I don't brought my computer :(